test_docstrings fails on i586 (Linux/openSUSE)

**Environment**
Python version: Python 3.8.2
Operation system: Linux/openSUSE/Tumbleweed (while building the official package for the distribution)
Architecture: i586

```
[   66s] _______________________________ test_docstrings ________________________________
[   66s] 
[   66s]     def test_docstrings():
[   66s]         """Test that all our docstrings are error-free."""
[   66s]         import enchant
[   66s]         import enchant.utils
[   66s]         import enchant.pypwl
[   66s]         import enchant.tokenize
[   66s]         import enchant.tokenize.en
[   66s]         import enchant.checker
[   66s]         import enchant.checker.CmdLineChecker
[   66s]     
[   66s]         try:
[   66s]             import enchant.checker.GtkSpellCheckerDialog
[   66s]         except ImportError:
[   66s]             pass
[   66s]         try:
[   66s]             import enchant.checker.WxSpellCheckerDialog
[   66s]         except ImportError:
[   66s]             pass
[   66s]         errors = []
[   66s]         #  Naive recursion here would blow the stack, instead we
[   66s]         #  simulate it with our own stack
[   66s]         tocheck = [enchant]
[   66s]         checked = []
[   66s]         while tocheck:
[   66s]             obj = tocheck.pop()
[   66s]             checked.append(obj)
[   66s]             newobjs = list(_check_docstrings(obj, errors))
[   66s]             tocheck.extend([obj for obj in newobjs if obj not in checked])
[   66s] >       assert not errors
[   66s] E       AssertionError: assert not [(<function set_prefix_dir at 0xb6c13d18>, 'plugins', 55)]
[   66s] 
[   66s] tests/test_docstrings.py:105: AssertionError
[   66s] ======================== 1 failed, 53 passed in 18.23s =========================
```

**Steps to Reproduce**
Please specify steps to reproduce the incorrect behaviour: just run the testsuite

The failure is not 100%, happens like one out of three attempts (?), more in the openSUSE build system (less memory) than on the local machine.

**Expected behavior**
Please tell us what you expected to happen: all tests pass

**Additional context**
If applicable, post the output of :

```python
{'provider': <Enchant: Hunspell Provider>, 'tag': 'en_US', '_broker': None,
'_this': None, '_Dict__describe_result': ('en_US', 'hunspell', 'Hunspell
Provider', '/usr/lib/enchant-2/enchant_hunspell.so')}
```

[Complete build log](https://github.com/pyenchant/pyenchant/files/4663432/_log.txt)

--- a/enchant/__init__.py
+++ b/enchant/__init__.py
@@ -906,6 +906,8 @@ def set_prefix_dir(path):
     """
     return _e.set_prefix_dir(path)
 
+    set_prefix_dir._DOC_ERRORS = ["plugins"]
+
 
 def get_user_config_dir():
     """Return the path that will be used by some

--- a/tests/test_checker.py
+++ b/tests/test_checker.py
@@ -205,11 +205,11 @@ def test_pwl():
 
 def test_bug2785373():
     """Testcases for bug #2785373."""
-    c = SpellChecker(enchant.Dict("en"), "")
+    c = SpellChecker(enchant.Dict("en_US"), "")
     c.set_text("So, one dey when I wes 17, I left.")
     for err in c:
         pass
-    c = SpellChecker(enchant.Dict("en"), "")
+    c = SpellChecker(enchant.Dict("en_US"), "")
     c.set_text("So, one dey when I wes 17, I left.")
     for err in c:
         pass

--- a/tests/test_dict.py
+++ b/tests/test_dict.py
@@ -128,7 +128,7 @@ def test_default_lang(en_us_dict):
 
 def test_pickling(en_us_dict):
     """Test that pickling doesn't corrupt internal state."""
-    d1 = Dict("en")
+    d1 = Dict("en_US")
     assert d1.check("hello")
     d2 = pickle.loads(pickle.dumps(d1))
     assert d1.check("hello")

--- a/tests/test_docstrings.py
+++ b/tests/test_docstrings.py
@@ -111,7 +111,7 @@ def _check_docstrings(obj, errors):
     if hasattr(obj, "__doc__"):
         skip_errors = [w for w in getattr(obj, "_DOC_ERRORS", [])]
         chkr = enchant.checker.SpellChecker(
-            "en", obj.__doc__, filters=[enchant.tokenize.URLFilter]
+            "en_US", obj.__doc__, filters=[enchant.tokenize.URLFilter]
         )
         for err in chkr:
             if len(err.word) == 1:

